### ISSUE TITLE
[Bug]: 4.66.0 breaks aws_appautoscaling_target

### ISSUE THREAD

**Issue by forma-wolf** [hashicorp/terraform-provider-aws#31227]
### Terraform Core Version

1.2.2

### AWS Provider Version

4.66.0

### Affected Resource(s)

aws_appautoscaling_target

### Expected Behavior

Tags should not be required on this resource.

### Actual Behavior

Tags seem to be required on the resource.

### Relevant Error/Panic Output Snippet

```shell
│ Error: listing tags for Application Auto Scaling Target (): InvalidParameter: 1 validation error(s) found.
│ - minimum field size of 1, ListTagsForResourceInput.ResourceARN.
```

### Terraform Configuration Files

resource "aws_appautoscaling_target" "appautoscaling_target" {
    max_capacity = var.max_capacity
    min_capacity = var.min_capacity
    resource_id = var.resource_id
    role_arn = var.role_arn
    scalable_dimension = var.scalable_dimension
    service_namespace = var.service_namespace
    tags = var.tags
}

### Steps to Reproduce

Create resource in previous provider version.
Remove terraform init config. 
Run terraform init again.
Run terraform plan.

### Debug Output

│ Error: listing tags for Application Auto Scaling Target (): InvalidParameter: 1 validation error(s) found.
│ - minimum field size of 1, ListTagsForResourceInput.ResourceARN.

### Would you like to implement a fix?

No

**Comment by jcave-relay** [comment]
Hi just leaving an FYI that the fix is being released as we speak in version 4.66.1

**Comment by ewbankkit** [comment]
@forma-wolf Thanks for raising this issue.
It has already been noticed in #31180. I'm going to close this one as a duplicate so that we can concentrate discussion in the linked issue.
Please add any additional comments there.
